src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipRequestValidator/WhenValidatingExpectedStartDate.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Esfa.Vacancy.Application.Commands.CreateApprenticeship;
+using Esfa.Vacancy.Domain.Validation;
+using FluentAssertions;
+using NUnit.Framework;
+using Ploeh.AutoFixture;
+
+namespace Esfa.Vacancy.UnitTests.CreateApprenticeship.Application.GivenACreateApprenticeshipRequestValidator
+{
+    [TestFixture]
+    public class WhenValidatingExpectedStartDate : CreateApprenticeshipRequestValidatorBase
+    {
+        private static List<TestCaseData> TestCases => new List<TestCaseData>
+        {
+            new TestCaseData(null, DateTime.Today.AddDays(1), false,
+                    new List<string> { ErrorCodes.CreateApprenticeship.ExpectedStartDateRequired },
+                    new List<string> { "'Expected Start Date' should not be empty." })
+                .SetName("And is null Then is invalid"),
+            new TestCaseData(DateTime.Today.AddDays(3), DateTime.Today.AddDays(4), false,
+                    new List<string> { ErrorCodes.CreateApprenticeship.ExpectedStartDateBeforeClosingDate },
+                    new List<string> { "'Expected Start Date' must be after the specified closing date." })
+                .SetName("And is before closing date Then is invalid"),
+            new TestCaseData(DateTime.Today.AddDays(3), DateTime.Today.AddDays(3), false,
+                    new List<string> { ErrorCodes.CreateApprenticeship.ExpectedStartDateBeforeClosingDate },
+                    new List<string> { "'Expected Start Date' must be after the specified closing date." })
+                .SetName("And is on closing date Then is invalid"),
+            new TestCaseData(DateTime.Today.AddDays(3).AddHours(12), DateTime.Today.AddDays(3), false,
+                    new List<string> { ErrorCodes.CreateApprenticeship.ExpectedStartDateBeforeClosingDate },
+                    new List<string> { "'Expected Start Date' must be after the specified closing date." })
```

Comment:
Perhaps add a case where ApplicationClosingDate is null

Reply:
null DateTime ends up being DateTime.MinValue, which then passes this test as 
1. we are only validating ExpectedStartDate (so we don't get a validation error for bad closing date)
1. the closing date is DateTime.MinValue which is before the ExpectedStartDate, so passes that rule.